WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -39,6 +39,16 @@ final class PaymentCaptureOrchestrator {
 
         ServiceLocator.stores.dispatch(action)
     }
+
+    func saveReceipt(for order: Order, params: CardPresentReceiptParameters) {
+//        let action = ReceiptAction.print(order: order, parameters: params)
```

Comment:
Did you mean to leave these commented-out lines here?

Reply:
Nope. I did not 😊